index.html

```diff
@@ -3,31 +3,71 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Quiz App</title>
+    <title>Quiz-App</title>
     <link rel="stylesheet" href="styles/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.1/css/all.css"
+      integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp"
+      crossorigin="anonymous"
+    />
   </head>
 
   <body class="app-grid">
     <header class="app-grid__header">
```

Comment:
If the `app-grid__header` doesn't have any styles which are dependent on the `app-grid` you can rename the class to `header`and start a new BEM block.
```suggestion
    <header class="header">
```